source/wrangling.md

```diff
@@ -843,7 +676,7 @@ tidy_lang
 ```
```

Comment:
again, lots of unexplained stuff going on above. `rename` and `drop`, `concat`, etc.

If we can simplify this code, we should

If we cannot (or can only to a small extent), we need to explain everything that is going on here in far more detail

Reply:
It's often a good idea to "look at the output" of a partial operation, go "oh man there's something wrong now!" and then fix it. This is way less mystical for students, because they see individually why each step is happening.

Reply:
I broke this up into a few steps. See if you think that is an improvement

Reply:
:+1: I like your suggestion @joelostblom: please feel free to change